types/recharts/index.d.ts

```diff
@@ -737,7 +745,7 @@ export interface XAxisProps {
 	tick?: boolean | any | React.ReactElement<any>;
 	mirror?: boolean;
 	reversed?: boolean;
-	label?: string | number | React.ReactElement<any>;
+	label?: string | number | React.ReactElement<any> | XLabel;
```

Comment:
I can't see any evidence in Recharts source code that supports bringing in an `XLabel` here. Could you please provide me with a reference to the Recharts source code?

Reply:
https://github.com/recharts/recharts/blob/master/src/component/Label.js